utils/Rector/UnderscoreToCamelCaseVariableNameRector.php

```diff
@@ -39,16 +39,12 @@ final class UnderscoreToCamelCaseVariableNameRector extends AbstractRector
      */
     private const PARAM_NAME_REGEX = '#(?<paramPrefix>@param\s.*\s+\$)(?<paramName>%s)#ms';
 
-    private ReservedKeywordAnalyzer $reservedKeywordAnalyzer;
-    private PhpDocInfoFactory $phpDocInfoFactory;
     private bool $hasChanged = false;
 
     public function __construct(
-        ReservedKeywordAnalyzer $reservedKeywordAnalyzer,
-        PhpDocInfoFactory $phpDocInfoFactory
+        private readonly ReservedKeywordAnalyzer $reservedKeywordAnalyzer,
+        private PhpDocInfoFactory $phpDocInfoFactory
```

Comment:
readonly is ok here:

```suggestion
        private readonly PhpDocInfoFactory $phpDocInfoFactory
```

Reply:
Thanks! Done.